.changeset/spotty-gifts-play.md

```diff
@@ -0,0 +1,6 @@
+---
+"hardhat": patch
```

Comment:
@alcuadrado not sure about this. Should it be a major?

Reply:
see the other comment